### ISSUE TITLE
Chat Complete with ChatGPT 3.5 and 4.0

### ISSUE THREAD

**Issue by peepsbill** [SashenJayathilaka/Tesla_Clone#13]
It seems the implementation uses create and does not currently support chat completion. Are there any plans to address this functionality?

**Comment by SashenJayathilaka** [comment]
Yes, this application has an error with **Request failed with status code 429**. I found some solutions on stackoverflow, but I can't find a really good solution for this error.

you can refer visiting below link.
[stackoverflow.](https://stackoverflow.com/questions/75041580/openai-api-giving-error-429-too-many-requests)
